src/main/java/uk/gov/pay/api/model/search/card/SearchRefunds.java

```diff
@@ -0,0 +1,119 @@
+    private static final String REFUNDS_PATH = "/v1/refunds";
+
+    public SearchRefunds(Client client,
+                         PublicApiConfig configuration,
+                         ConnectorUriGenerator uriGenerator,
+                         ObjectMapper objectMapper) {
+        super(client, configuration, uriGenerator, objectMapper);
+    }
+
+    public Response getSearchResponse(Account account, Map<String, String> queryParams) {
+        validateSupportedSearchParams(queryParams);
+
+        String url = connectorUriGenerator.refundsURIWithParams(account, queryParams);
+        Response connectorResponse = client
+                .target(url)
+                .request()
+                .header(HttpHeaders.ACCEPT, APPLICATION_JSON)
+                .get();
+        logger.info("response from connector for refunds search: " + connectorResponse);
+        if (connectorResponse.getStatus() == SC_OK) {
+            return processResponse(account.getAccountId(), connectorResponse);
+        }
+        throw new SearchRefundsException(connectorResponse);
+    }
+
+    private Response processResponse(String accountId, Response connectorResponse) {
+        try {
+            JsonNode responseJson = connectorResponse.readEntity(JsonNode.class);
+            TypeReference<SearchRefundsResponse> typeRef = new TypeReference<SearchRefundsResponse>() {
+            };
+            SearchRefundsResponse searchResponse = objectMapper.readValue(responseJson.traverse(), typeRef);
+            List<RefundForSearchRefundsResult> results = searchResponse.getRefunds()
+                    .stream()
+                    .map(refund -> RefundForSearchRefundsResult.valueOf(refund))
+                    .collect(Collectors.toList());
+
+            HalRepresentation.HalRepresentationBuilder halRepresentation = HalRepresentation
+                    .builder()
+                    .addProperty("results", results);
+            return Response.ok().entity(decoratePagination(Long.valueOf(accountId), halRepresentation, searchResponse)
+                    .build().toString())
+                    .build();
+        } catch (IOException | ProcessingException ex) {
+            throw new SearchRefundsException(ex);
+        }
+    }
+
+    private HalRepresentation.HalRepresentationBuilder decoratePagination(Long accountId,
+                                                                          HalRepresentation.HalRepresentationBuilder halRepresentationBuilder,
+                                                                          ISearchPagination pagination) {
+        HalRepresentation.HalRepresentationBuilder builder = addPaginationProperties(halRepresentationBuilder, pagination);
+        SearchRefundsNavigationLinks links = (SearchRefundsNavigationLinks) pagination.getLinks();
+        String path = getRefundsPath(accountId);
+
+        try {
+            addLink(builder, "self", transformIntoPublicUri(baseUrl, links.getSelfPage(), path));
```

Comment:
I think `path` should just be `REFUNDS_PATH`, which is  "/v1/refunds". We don't need `/accounts/accountid` in publicapi as the account depends on the bearer token, you can't retrieve resources for other accounts